pennylane/transforms/optimization/single_qubit_fusion.py

```diff
@@ -109,7 +130,7 @@ def qfunc(r1, r2):
             next_gate_idx = find_next_gate(current_gate.wires, list_copy[1:])
 
         # Only apply if the cumulative angle is not close to 0
-        if not allclose(cumulative_angles, zeros(3), atol=tol):
+        if not allclose(cumulative_angles, zeros(3), atol=atol):
```

Comment:
Do you want to set `rtol=0` here? Otherwise, there remains a relative tolerance check in addition to the absolute tolerance check happening behind the scenes

Reply:
Alternatively, you could allow the user to pass `atol` _and_ `rtol` as keyword arguments for full control 😆

Reply:
Let's set `rtol=0`. I'm just thinking intuitively, I would be looking for absolute closeness to 0.